website/docs/r/subnet.html.markdown

```diff
@@ -85,7 +85,7 @@ A `service_delegation` block supports the following:
 
 * `name` - (Required) The name of service to delegate to. Possible values include: `Microsoft.BareMetal/AzureVMware`, 
 `Microsoft.BareMetal/CrayServers`, `Microsoft.Batch/batchAccounts`, `Microsoft.ContainerInstance/containerGroups`, `Microsoft.Databricks/workspaces`, `Microsoft.HardwareSecurityModules/dedicatedHSMs`, `Microsoft.Logic/integrationServiceEnvironments`, `Microsoft.Netapp/volumes`, `Microsoft.ServiceFabricMesh/networks`, `Microsoft.Sql/managedInstances`, `Microsoft.Sql/servers`, `Microsoft.Web/hostingEnvironments` or `Microsoft.Web/serverFarms`.
-* `actions` - (Optional) A list of Actions which should be delegated. Possible values include: `Microsoft.Network/virtualNetworks/subnets/action`.
+* `actions` - (Required) A list of Actions which should be delegated. Required values are: `Microsoft.Network/virtualNetworks/subnets/join/action ,  Microsoft.Network/virtualNetworks/subnets/prepareNetworkPolicies/action`.
```

Comment:
AFAIK it's possible for users to pick and choose these values, rather than requiring all of them to be set?

```suggestion
* `actions` - (Required) A list of Actions which should be delegated. Possible values are: `Microsoft.Network/virtualNetworks/subnets/join/action ,  Microsoft.Network/virtualNetworks/subnets/prepareNetworkPolicies/action`.
```

Reply:
The only thing I have to compare with is that if the delegation is done manually in the webportal then you get this result. I don't know if there are other possible actions, but these two are required (at least this week....) for the delegation to work.

Reply:
since it's possible for users to pick from one of these values ([including the original one](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/examples/VirtualNetworkGetWithServiceAssociationLink.json#L56)) could we add this old value back in/update the language here?